### DISCUSSION TITLE
too fast QBO in EAMv2

### DISCUSSION THREAD

**Discussion by wenyuz** [E3SM-Project/E3SM#4685] [E3SM model help]
Hi,

I ran the EAMv2 with an F2000 setup and found that the QBO is too fast (with a period of half-year).

I notice that EAMv2 has already incorporated the parameter tuning for better QBO, as suggested by Richter et al. 2019.

In Richter et al. 2019 (based on EAMv1), they reduces the values of the following two parameters: effgw_beres = 0.35 (default 0..4 in EAMv1) and gw_convect_hcf=10 (default 20 in EAMv1). This reduces the magnitude of the gravity momentum source and consequently makes the QBO less fast in EAMv1.

I tried to further reduce the values of these two parameters (effgw_beres = 0.1 and gw_convect_hcf=4) but the QBO period does not change much and is still at about half year.

I wonder if there is any explanation for this behavior of EAMv2.

Thanks,

Richter et al. 2019: Improved Simulation of the QBO in E3SMv1. JAMES.

Wenyu Zhou

**Comment by whannah1** [comment]
@wenyuz my only thought on this is that the DCAPE trigger modification in the Zhang-McFarlane (ZM) deep convection scheme that was added for v2 will effectively reduce the amount of convective heating that is used for gravity wave generation, which _might_ explain why you're seeing a different sensitivity to changing those parameters. 

Coincidently, I just made this figure the other day directly comparing the precipitation in v1 and v2:

![image](https://user-images.githubusercontent.com/9679059/144464330-61f3a10a-3de8-435f-b70c-bb90aeafeba6.png)

This is a 2 year average of 3 precip variables - total, convective, and "large-scale" - of some coupled runs I'm doing that branch from the 500-year pre-industrial control simulations for v1 and v2. I was just making this plot as a sanity check that things were looking reasonable, but it's quite striking how the convective precipitation reduces in v2. I think "convective" precip includes precip from ZM and CLUBB, but I suspect the majority of this difference is due to the DCAPE change in ZM. 

One way you might be able to directly test this is to artificially enhance the convective heating that is used for gravity waves.
The way I would try to do this is to take the "TTEND_DP" variable that is set here:
`components/eam/src/physics/cam/convect_deep.F90`
and just multiply it by some positive constant greater than 1, maybe 2. 
This data is the heating tendency that is used to estimate the convective gravity wave source, so if the reduction of heating from ZM is the problem, then artificially enhancing this tendency we should see stronger gravity wave tendencies in the tropics, and perhaps a noticeable change in the QBO phase speed.

**Reply by wenyuz** [reply]
Hi Walter,

Thank you for your insights. I also feel the switch to dCAPE/dt closure is a big change and could affect many things.

However, as I understand, if there is less convective heating to trigger gravity wave momentum flux, the QBO downward propagation should be slower instead of faster. In other words, enhancing the convective hearting would make the QBO propagation even faster.

The QBO was too fast in EAMv1, so Richter et al. 2019 reduced the efficiency of the convective heating to trigger gravity wave by tuning the following two parameters: effgw_beres = 0.35 (default 0.4 in EAMv1) and gw_convect_hcf=10 (default 20 in EAMv1). This slows down the propagation of the QBO and makes it more realistic.

The parameter tuning of Richter et al. 2019 has been adopted in the default EAMv2 namelist. However, QBO in the EAMv2 is still too fast. I actually tried further reducing the parameters with effgw_beres = 0.1 and gw_convect_hcf=4, but somehow the QBO period was not affected. So I am quite puzzled.

Thanks,

Wenyu

**Reply by whannah1** [reply]
Oh I see, then maybe we need to directly diagnose the climatology of the heating tendency that's used for the convective gravity wave scheme. Perhaps the DCAPE trigger is making the convective heating less frequent, but with larger tendencies when they do fire, which could culminate in a larger net gravity wave momentum transport over a long time?

Either way, I think it's likely that DCAPE is key to the explanation, so maybe the easier thing to try is disabling it and seeing how the QBO responds.

**Reply by wenyuz** [reply]
Thanks for your suggestion. I will run a case with conventional CAPE closure and see how it affects the QBO.

**Reply by whannah1** [reply]
Just so you know, the DCAPE option affects the "trigger" and not the "closure".

**Reply by wenyuz** [reply]
Oh! So the DCAPE_trigger option only controls when will the deep convection trigger, and the cloud base mass flux is still computed in the conventional way of (CAPE-CAPE0)/tau?

**Reply by whannah1** [reply]
Exactly. There is also the "unrestricted launch level" (ULL) modification that changes "where" the cloud base occurs. I'm not sure how that might play a role here. The ULL change probably doesn't have much affect over tropical oceans, it was moreso motivated by the elevated convection that is sometimes observed over land at night. But this also shouldn't fundamentally change the basic behavior of the closure.

**Reply by wenyuz** [reply]
I see. Thank you so much for your explanation. Good to know that DCAPE only affects the trigger.

**Comment by wenyuz** [comment]
Here are some results with tuned parameters. What I saw in the control EAMv2 may not be QBO but semi-annual oscillation. In other words, the QBO is absent in the control EAMv2 simulation.

I further tuned the parameters related to the convective gravity drag but still could not get a reasonable QBO. Here are my results. The closest case is with effgw_beres = 0.4; gw_convect_hcf = 20, which produces an ~annual oscillation.

![image](https://user-images.githubusercontent.com/14170302/145456284-aaf9998f-33cb-4e1e-9b6f-8ea74e878fa2.png)

![image](https://user-images.githubusercontent.com/14170302/145456325-59ab08f3-54d8-4e4f-ba2a-0620910e2c58.png)

![image](https://user-images.githubusercontent.com/14170302/145456357-54096ec1-ab71-4604-987d-e6fedbc6d657.png)

![image](https://user-images.githubusercontent.com/14170302/145456416-a5541901-5c33-4aa3-9601-7b45f158a98d.png)

![image](https://user-images.githubusercontent.com/14170302/145456428-19e9067c-2c14-4163-b0eb-36320a7f3476.png)

![image](https://user-images.githubusercontent.com/14170302/145456454-abd74b29-187f-422a-bf97-56d5727eeece.png)

![image](https://user-images.githubusercontent.com/14170302/145456480-8ed9cb42-e618-4da1-a282-c1b09073498e.png)

**Reply by whannah1** [reply]
Thanks for posting this analysis. It seems my inclination to blame DCAPE was wrong, which is good to know. 
Any other ideas to explain the lack of QBO in v2?

**Reply by wenyuz** [reply]
I only tested trigger_dcape_ull = false with effgw_beres=0.35 and convect_hcf=10. It is possible that trigger_dcape_ull = false may play a larger role when applied with other values of effgw_beres and convect_hcf. I will test that further.

But, on the other hand, I do not want to change the basic setup of EAMv2 (i.e., set trigger_dcape_ull = false) when I tune the QBO.

**Comment by whannah1** [comment]
@wenyuz Have you tried disabling `use_gw_energy_fix` in your v2 experiments?

**Reply by wenyuz** [reply]
Yes, I actually tried one case with use_gw_energy_fix = .false. but It did not help much.

![image](https://user-images.githubusercontent.com/14170302/146048833-f9bb493a-e3ac-435f-8d5e-d61026b288e4.png)

**Comment by wenyuz** [comment]
I had some discussions with Yaga Richter. She told me that, in the historical simulation when they tuned effgw_beres=0.35 and convect_hcf=10, EAMv2 was able to simulate a QBO with a ~2 year period but with a very weak amplitude.

https://web.lcrc.anl.gov/public/e3sm/diagnostic_output/wlin/E3SMv2/v2.LR.HIST_ENS/e3sm_diags/180x360_aave/model_vs_obs_1985-2014/viewer/qbo/variable/qbo-era-interim/plot.html
![image](https://user-images.githubusercontent.com/14170302/146049884-f77ae021-455b-4dee-8bca-0a73c742c6fa.png)

What I ran is the AMIP F2000. I use the newest code on this GitHub page and the default run does not change any parameter.

It will be surprising if the difference between AMIP and coupled simulations is so large. So I wonder if there may be some code differences between them. Thanks.
